libs/pdi-scanner/src/ts/scanner_client.ts

```diff
@@ -42,11 +45,11 @@ export interface ScannerStatus {
   /**
    * @deprecated Not used by PageScan 6, always false.
    */
-  frontM5SensorCovered: never;
+  frontM5SensorCovered: false;
   /**
    * @deprecated Not used by PageScan 6, always false.
    */
-  frontRightSensorCovered: never;
+  frontRightSensorCovered: false;
```

Comment:
Turns out there's no way to construct a valid object literal if you have required fields with the `never` type